src/routes/tools/extreme-precipitation/_ExploreData.svelte

```diff
@@ -0,0 +1,253 @@
+    slugs = [],
+    content = "",
+    header = "Glossary",
+  }) {
+    learnMoreProps = { slugs, content, header };
+    showLearnMore = true;
+    LearnMoreModal = (
+      await import(
+        "~/components/tools/Partials/LearnMore/LearnMoreModal.svelte"
+      )
+    ).default;
+  }
+
+  async function loadShare() {
+    if ($boundary.id === "custom") {
+      bookmark = "Cannot create a bookmark for an uploaded boundary";
+    } else {
+      const [lng, lat] = $location.center;
+      const modelsStr = $modelsStore.join(",");
+      bookmark = `climvar=${$climvarStore}&scenario=${$scenarioStore}&models=${modelsStr}&lng=${lng}&lat=${lat}&boundary=${$boundary.id}`;
+    }
+    showShare = true;
+    ShareLink = (await import("~/components/tools/Partials/ShareLink.svelte"))
+      .default;
+  }
+
+  async function loadLocation() {
+    showChangeLocation = true;
+    ChangeLocation = (
+      await import("~/components/tools/Partials/ChangeLocationStation.svelte")
+    ).default;
+  }
+
+  function formatDailyDataForExport(_arr) {
+    return _arr.map((item) => {
+      const row = {};
+      row.date = item.date;
+      item.values.forEach((d) => {
+        if (Array.isArray(d.value)) {
+          row[`${d.label} Min`] = d.value[0];
+          row[`${d.label} Max`] = d.value[1];
+        } else {
+          row[d.label] = d.value;
+        }
+      });
+      return row;
+    });
+  }
+
+  async function loadDownload() {
+    showDownload = true;
+    if ($indicator.id === "timing") {
+      csvData = formatDailyDataForExport(dataByDate);
+    } else {
+      csvData = formatDataForExport(dataByDate);
+    }
+    metadata = [
+      ["boundary", $boundary.id],
+      ["feature", $location.title],
+      ["center", `${$location.center[0]}, ${$location.center[1]}`],
+      ["scenario", $scenario.label],
+      ["climate indicator", `${$climvar.label} ${$indicator.label}`],
+      ["threshold", `${$thresholdStore} ${$climvar.units.imperial}`],
+    ];
+    printContainer = document.querySelector("#explore-data");
+    printSkipElements = ["settings"];
+    DownloadChart = (
+      await import("~/components/tools/Partials/DownloadChart.svelte")
+    ).default;
+  }
+
+  function changeLocation(e) {
+    if (e.detail.boundaryId === "custom") {
+      locationStore.updateBoundary("locagrid");
+      locationStore.updateLocation(e.detail.location, true);
+    } else {
+      locationStore.updateBoundary(e.detail.boundaryId);
+      locationStore.updateLocation(e.detail.location);
+    }
+  }
+</script>
+
+{#if $isFetchingStore}
+  <Loading />
+{/if}
+
+<Dashboard>
+  <div slot="chart_title" class="block title">
+    <ChartTitle
+      title="{chartTitle}"
+      indicatorLabel="{indicatorLabel}"
+      durationLabel="{durationLabel}"
+      scenarioLabel="{$scenario.labelLong}"
+      thresholdLabel="{thresholdLabel}"
+      intervalsLabel="{intervalsLabel}"
+      loadLocation="{loadLocation}"
+    />
+  </div>
+
+  <div slot="stats">
```

Comment:
Stats is not yet implemented. Will need to brainstorm what stats to show for the Intensity chart.